services/tenant-ui/frontend/src/components/common/LocaleSwitcher.vue

```diff
@@ -9,7 +9,7 @@ import { I18nLocale } from '@/helpers';
 
 const { locale } = useI18n({ useScope: 'global' });
 
-const localeList: Array<I18nLocale> = ['en', 'fr', 'jp'];
+const localeList: Array<I18nLocale> = ['en', 'fr', 'ja'];
```

Comment:
Just a note: this might be something we want to look at having dynamically populated based on the locales actually available - if possible.